pkgs/development/python-modules/scapy/default.nix

```diff
@@ -24,8 +25,20 @@ buildPythonPackage rec {
     sha256 = "0nxci1v32h5517gl9ic6zjq8gc8drwr0n5pz04c91yl97xznnw94";
   };
 
+  patches = [
+    ./find-library.patch
```

Comment:
why not
```suggestion
    (substituteAll {
      src = ./find-library.patch;
      libpcap_file = "${lib.getLib libpcap}/lib/libpcap${stdenv.hostPlatform.extensions.sharedLibrary}";
    })
```

Reply:
and postPatch logic should go away

Reply:
> why not [...]

Because I want to check at build time that the library path exists. (But I can change it if you insist.)

Reply:
@jonringer : OK for merge?

Reply:
The lib will always existe and the patch will fail to apply.

Reply:
?

Reply:
The check in you postPatch should always be useless since this package take libpcap as input. It is fine to be safer. I don't know if code change and the patch can not be applied if the build fail. If it fail then it is 100% dead code.